docs/victoriametrics/vmbackupmanager.md

```diff
@@ -460,6 +460,7 @@ command-line flags:
      Disable weekly run. Default false
   -dst string
      The root folder of Victoria Metrics backups. Example: gs://bucket/path/to/backup/dir, s3://bucket/path/to/backup/dir or fs:///path/to/local/backup/dir
+     Note: In case of GS/S3 storage, URL should not contain hostname and only name of the bucket
```

Comment:
Changes in command-line flags description require changes in related Go code. Also, this is relevant to vmbackup and vmrestore as well, so it would be great to cover all of these.

Also, this is only relevant to S3 storage as GCS does not support custom endpoints. Here is a rephrased version of this one:
```
The hostname of the S3 server must be specified via the -customS3Endpoint command-line flag. See https://docs.victoriametrics.com/victoriametrics/vmbackup/#using-custom-s3-endpoint for more details.
```